lib/record_store/provider/dnsimple.rb

```diff
@@ -4,7 +4,7 @@ module RecordStore
   class Provider::DNSimple < Provider
     class << self
       def record_types
-        super | Set.new(['SSHFP'])
+        super | Set.new(['PTR', 'SSHFP'])
```

Comment:
syntactic preference on my part to use `%w(PTR SSHFP)` for things like this... I can see how this could be inconsistent with other code in this repo

Reply:
Switched over to the alternate array literal syntax =)